src/modules/colorPicker/ColorPickerUI/Views/ColorEditorView.xaml

```diff
@@ -284,20 +301,18 @@
             BorderBrush="{DynamicResource CardStrokeColorDefaultBrush}"
             BorderThickness="0,1,0,0"
             Opacity="0">
-            <Border.Background>
-                <SolidColorBrush Color="{DynamicResource SystemAccentColorPrimary}" />
-            </Border.Background>
             <StackPanel
                 HorizontalAlignment="Center"
                 VerticalAlignment="Center"
                 AutomationProperties.LiveSetting="Assertive"
                 AutomationProperties.Name="{x:Static p:Resources.Copied_to_clipboard}"
                 Orientation="Horizontal">
-                <ui:SymbolIcon
+                <TextBlock 
+                    FontFamily="Segoe Fluent Icons, Segoe MDL2 Assets"
```

Comment:
Tip: never hardcode values like this. This is super sensitive to typos and other reasons for failure. Use a FontIcon instead :)

Reply:
There is no FontIcon in .net WPF.

Reply:
I've made the change.